### PR TITLE
Extend model objects with CustomStringConvertible

### PR DESCRIPTION
**Pull request by larryonoff**

### CODE REVIEW

**Comment by Skifcha** [comment]
@larryonoff thanks, we'll get back to it asap. ETA – beginning of the next week.

**Comment by larryonoff** [comment]
@Skifcha sure. will do.

**Comment by larryonoff** [comment]
@Skifcha done

File: Adapty/Classes/CustomStringConvertible.swift [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+            "price": price,
+            "currencyCode": currencyCode,
+            "currencySymbol": currencySymbol,
+            "regionCode": regionCode,
+            "subscriptionPeriod": subscriptionPeriod,
+            "introductoryDiscount": introductoryDiscount,
+            "subscriptionGroupIdentifier": subscriptionGroupIdentifier,
+            "discounts": discounts,
+            "localizedPrice": localizedPrice,
+            "discounts": discounts,
```
Comment (AKyashkin) [inline Adapty/Classes/CustomStringConvertible.swift:50]: This key is duplicated and should be removed.

**Review by AKyashkin — COMMENTED** [review]
This file is also should be added to the pod, right? Right now he's not even in complied sources.

**Comment by AKyashkin** [comment]
@larryonoff can you add this file to compiled source of the pod? The rest looks fine.

**Comment by larryonoff** [comment]
> @larryonoff can you add this file to SDK? The rest looks fine.

Sorry. I'm not sure I understand what you mean. 

_Adapty.podspec_ has pattern for source files in _Classes_ folder, so that _Package.swift_.

**Comment by AKyashkin** [comment]
The Project itself doesn't see this file in case of the dev pod. No worries, I'll add this small thing later.

**Pull request merged**